package.json

```diff
@@ -21,6 +21,8 @@
     "@angular/platform-browser-dynamic": "^6.0.0",
     "@angular/router": "^6.0.0",
     "core-js": "^2.5.4",
+    "ng2-filter-pipe": "^0.1.10",
+    "ng2-search-filter": "^0.4.7",
```

Comment:
Jak dla mnie są to na tyle proste funkcjonalności, że nie potrzebujemy do nich dodatkowych importów. Zwłaszcza tych :) o ile drugi z nich jest w miarę bo ma dużo pobrań na NPM i gwiazdek na Githubie to ten pierwszy nie ma ich praktycznie w ogóle a to oznacza, że będzie miał zerowe wsparcie - nie chcemy używać takich rzeczy.

Spróbuj więc zaimplementować to samemu.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/8f172e9866f6718d276121c112003a5f00026887